src/Asymmetric/Crypto.php

```diff
@@ -1,20 +1,12 @@
 <?php
-declare(strict_types=1);
+declare(strict_types = 1);
 namespace ParagonIE\Halite\Asymmetric;
 
-use ParagonIE\Halite\Alerts\{
-    CannotPerformOperation,
-    InvalidKey,
-    InvalidMessage,
-    InvalidSignature
-};
 use ParagonIE\Halite\{
-    Util as CryptoUtil,
-    Halite,
-    HiddenString,
-    Key,
-    Symmetric\Crypto as SymmetricCrypto,
-    Symmetric\EncryptionKey
+    Halite, HiddenString, Key, Symmetric\Crypto as SymmetricCrypto, Symmetric\EncryptionKey, Util as CryptoUtil
```

Comment:
@sstok  These imports usually will be eclipsed and that's why I make these lines join to one line.